main.js

```diff
@@ -0,0 +1,234 @@
+        console.log("array two wins check successful")
+        document.getElementById('statusBoard').innerHTML = "Player " + playerTurn + "WINS!!!"
+    }
+}
+
+//--------------------function to change cell status, place x or o---------//
+function cellMove(cellID) {
+    console.log(playerTurn)
+    console.log(cellID)
+    if (cellLookup[cellID].takenO === true || cellLookup[cellID].takenX === true) {
+        document.getElementById('statusBoard').innerHTML = "That square is taken! Try a different move."
+    } else if (playerTurn === "playerO") {
+        cellLookup[cellID].takenO = true; //sets taken status to true
+        document.getElementById(cellLookup[cellID].ID).innerHTML = "O";
+        document.getElementById('statusBoard').innerHTML = ("Player O took cell " + cellLookup[cellID].ID);
+        playerOList.push(cellLookup[cellID].ID)
+        winCheck(playerOList);
+        playerTurn = "playerX" //switches player turn to other player
+        console.log("inside else if " + playerTurn)
+        document.getElementById('whoseTurn').innerHTML = "It's " + playerTurn + "'s turn."
+        console.log("Player X moves: " + playerXList)
+        console.log("Player O moves: " + playerOList)
+    } else {
+        cellLookup[cellID].takenX = true;
+        document.getElementById(cellLookup[cellID].ID).innerHTML = "X";
+        playerXList.push(cellLookup[cellID].ID) //adds player move to player move list
+        document.getElementById('statusBoard').innerHTML = ("Player X took cell " + cellLookup[cellID].ID);
+        winCheck(playerXList)
+        playerTurn = "playerO" //switches player turn to other player
+        console.log("inside else for " + playerTurn)
+        document.getElementById('whoseTurn').innerHTML = "It's " + playerTurn + "'s turn."
+        console.log("Player X moves: " + playerXList)
+        console.log("Player O moves: " + playerOList)
+    }
+}
+
+//----------------------event listeners for cell move function ------------//
+document.getElementById('cell-0').addEventListener('click', () => {
+    cellMove('cellZero');
+}, 2000)
+document.getElementById('cell-1').addEventListener('click', () => {
+    cellMove('cellOne');
+}, 2000)
+document.getElementById('cell-2').addEventListener('click', () => {
+    cellMove('cellTwo');
+}, 2000)
+document.getElementById('cell-3').addEventListener('click', () => {
+    cellMove('cellThree');
+}, 2000)
+document.getElementById('cell-4').addEventListener('click', () => {
+    cellMove('cellFour');
+}, 2000)
+document.getElementById('cell-5').addEventListener('click', () => {
+    cellMove('cellFive');
+}, 2000)
+document.getElementById('cell-6').addEventListener('click', () => {
+    cellMove('cellSix');
+}, 2000)
+document.getElementById('cell-7').addEventListener('click', () => {
+    cellMove('cellSeven');
+}, 2000)
+document.getElementById('cell-8').addEventListener('click', () => {
+    cellMove('cellEight');
+}, 2000)
```

Comment:
You could use a loop to iterate over your `cellLookup` object to add all of these event listeners without all the duplicate code.